docs/src/start/connect-azure.md

```diff
@@ -224,9 +224,9 @@ Alternatively, post your own custom messages on `az/messages/events/#` topic:
 tedge mqtt pub az/messages/events/ '{"text": "My message"}'
 ```
 
-To view the messages that were sent from the device to the cloud, follow this [document](https://docs.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-cli#create-and-monitor-a-device).
+To view the messages that were sent from the device to the cloud, follow this [document](https://learn.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-cli).
 
-More info about sending telemetry to Azure can be found [here](https://docs.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-dotnet)
+More info about sending telemetry to Azure can be found [here](https://learn.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-dotnet)
```

Comment:
This one redirected me to https://learn.microsoft.com/en-us/azure/iot/tutorial-send-telemetry-iot-hub?pivots=programming-language-csharp

Reply:
Similar comment as above. I think the main goal is to make sure we're on the correct subdomain. The extra query parameters look to be added dynamically by ms.

Reply:
Resolved by 383287ca8947bf1431366ae84c0af31518d63add